calico-enterprise/getting-started/upgrading/upgrading-enterprise/openshift-upgrade.mdx

```diff
@@ -146,6 +146,17 @@ The steps differ based on your cluster type. If you are unsure of your cluster t
    EOF
    ```
 
+1. If your cluster was at one time v3.18 or older, remove deprecated fields from the [Manager](../../../reference/installation/api.mdx#operator.tigera.io/v1.Manager).
+
+```bash
+oc apply -f - <<EOF
+apiVersion: operator.tigera.io/v1
+kind: Manager
+metadata:
+  name: tigera-secure
+EOF
+   ```
```

Comment:
For this to paginate correctly, the code block needs to align with the same column as the "I" in "If".